### DISCUSSION TITLE
Talk Talk

### DISCUSSION THREAD

**Discussion by markharrison** [markharrison/Playground#2] [Show and tell] [answered]
Well, did I tell you before when I was up?
Anxiety was bringing me down
I'm tired of listening to you talking in rhymes
Twisting around to make me think you're straight down the line

All you do to me is talk, talk
Talk, talk, talk, talk

**Comment by markharrison** [comment]
If every sign that I see is complete
Then I'm a fool in your game
And all you want to do is tell me your lies
Won't you show the other side, you're just wasting my time

All you do to me is talk, talk
Talk, talk, talk, talk

**Comment by markharrisonio** [comment, marked as answer]
All you do to me is talk, talk
Talk, talk, talk, talk
